titus-server-federation/src/main/java/com/netflix/titus/federation/endpoint/rest/FederationV2CapacityGroupResource.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.federation.endpoint.rest;
+
+import java.util.List;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+
+import com.netflix.titus.api.endpoint.v2.rest.representation.ApplicationSlaRepresentation;
+import com.netflix.titus.common.util.tuple.Either;
+import com.netflix.titus.federation.endpoint.EndpointConfiguration;
+import com.netflix.titus.federation.service.CellWebClientConnector;
+import com.netflix.titus.federation.service.CellWebClientConnectorUtil;
+import io.swagger.annotations.Api;
+import org.springframework.core.ParameterizedTypeReference;
+
+/**
+ * Supports subset of operations provided by {@link com.netflix.titus.grpc.protogen.SchedulerServiceGrpc}, applicable
+ * at the federation level.
+ */
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Api(tags = "CapacityManagement")
+@Path(FederationV2CapacityGroupResource.BASE_PATH)
+@Singleton
+public class FederationV2CapacityGroupResource {
+
+    public static final String BASE_PATH = "/v2/management/applications";
+
+    public static final String API_PATH = "/api/v2/management/applications";
+
+    private static final ParameterizedTypeReference<ApplicationSlaRepresentation> APPLICATION_SLA_TP =
+            ParameterizedTypeReference.forType(ApplicationSlaRepresentation.class);
+
+    private static final ParameterizedTypeReference<List<ApplicationSlaRepresentation>> APPLICATION_SLA_LIST_TP =
+            new ParameterizedTypeReference<List<ApplicationSlaRepresentation>>() {
+            };
+
+    private final EndpointConfiguration configuration;
+    private final CellWebClientConnector cellWebClientsSupplier;
+
+    @Inject
+    public FederationV2CapacityGroupResource(EndpointConfiguration configuration,
+                                             CellWebClientConnector cellWebClientsSupplier) {
```

Comment:
minor nit: rename variable and field, it looks like this was a `Supplier<...>` before?